src/molecules/Footers/OppskriftFooter.js

```diff
@@ -5,46 +5,31 @@ import { Col, Row, Grid } from '../..';
 import { SvgIcon } from '../../atoms/SvgIcon';
 
 const FooterWrapper = styled.footer`
-	background-color: ${props => props.theme.colors.darkGrey};
+	background-color: ${props => props.theme.colors.type};
 	color: ${props => props.theme.colors.white};
-	padding: 10.3rem 0 6.2rem;
 	font-family: ${props => props.theme.variables.mainFont};
-	font-size: 1.4rem;
-	letter-spacing: 2px;
-	
-	@media screen and (max-width: ${props => props.theme.flexboxgrid.breakpoints.md}rem) {
-		font-size: 3.2rem;
-		background-color: ${props => props.theme.colors.type};
-		padding-top: 14rem;
-		
-		.margin-bottom-sm-3{
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-5{
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-6{
-			margin-bottom: 3rem;
-		}
-		.margin-bottom-sm-10{
-			margin-bottom: 3rem;
-		}
+	font-size: ${props => props.theme.variables.uiRegularSize};
+	line-height: ${props => props.theme.variables.headingMediumLineHeight};
```

Comment:
the similarly named fontsizes and line-heights should go pretty well together. uiRegularSize should probably go with uiRegularLineHeight, and headingMediumLineHeight should probably go with headingMediumSize.

Is there anything special about this component that says we should deviate from that pattern?

Reply:
i tried uiRegularLineHeight at first, though it looks better with headingMediumLineHeight (according to designs)

Reply:
How about using headingSmallSize and headingSmallLineHeight?

If the font-size and line-height pair do not match as expected, we should look into if the values should be adjusted.

Reply:
hmm, i'm refactoring recipe page right now, gonna try different line height for uiRegular 👍